jennchenn/uwexplore
# login

## Implementation description

- call auth on login 
- save token in parent nav for other component use



## Steps to test

1. log in with existing account using front end
2. make sure no errors in the console



## What should reviewers focus on?

- sign up is stubbed in but has errors, will move to anther task so I can focus on other tickets with higher priority
